python/README.md

```diff
@@ -73,7 +73,7 @@ Setting `maxParallelForks` isn't required anymore since now Gradle parallelism i
 #### Generating reports
 
 ```shell script
-python/experiments/generate-box-tests-reports.main.kts
+JAVA_OPTS="-Xmx1G" python/experiments/generate-box-tests-reports.main.kts
```

Comment:
There is the same line in the end of this README file, please add the variable there too and merge